pkg/link/link_api.go

```diff
@@ -59,21 +58,57 @@ func getConfigureAPIKey() (string, error) {
 
 func getConfigureBaseURL() (string, error) {
 	reader := bufio.NewReader(os.Stdin)
-	fmt.Printf("Enter the base URL for this link [default: %s]: ", aurora.Bold(aurora.Cyan(defaultBaseURL)))
-	baseURL, err := reader.ReadString('\n')
+  fmt.Printf(
+  `Which base url should be used? Enter a number:
```

Comment:
Wording on this could be tweaked to try and make everything uniform (realise the current stuff isn't actually uniform!)

Potential suggestion: 
```
fmt.Printf(
`Select the base URL for this link [default: %s]:
  1. production
  2. staging
  3. test
  4. custom`, defaultSelectionCharacter)
```

Reply:
Yeah brilliant, I'll tidy this up on Tuesday when I'm back from holiday 👍